### PR TITLE
Racing game Armagetron Advanced added 

### PR DESCRIPTION
**Pull request by leenaars**

### CODE REVIEW

**Comment by leenaars** [comment]
When packaging the game Armagetron Advanced for my kid I noticed that many games were not in the appropriate section (under GAMES).

File: pkgs/games/armagetronad/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  };
+
+  NIX_LDFLAGS = "-lSDL_image";
+
+  configureFlags ="--disable-etc";
+  buildInputs = [SDL SDL_image libxml2 libjpeg mesa zlib];
+
+  meta = {
+    homepage = http://armagetronad.org;
+    description = "An multiplayer networked arcade racing game in 3D similar to Tron.";
```
Comment (pSub) [inline pkgs/games/armagetronad/default.nix]: `description` should not end with a period.

File: pkgs/games/armagetronad/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  versionMinor = "3.3";
+  version = "${versionMajor}.${versionMinor}";
+in
+
+stdenv.mkDerivation {
+  name = "armagetron-${version}";
+  src = fetchurl {
+    url = "https://launchpad.net/armagetronad/${versionMajor}/0.2.8.3.x/+download/armagetronad-${version}.src.tar.bz2";
+    sha256 = "1s55irhg60fpmhy8wwxpdq7c45r1mqch6zpicyb2wf9ln60xgwnx";
+
```
Comment (pSub) [inline pkgs/games/armagetronad/default.nix]: Please remove those blank lines.

**Comment by pSub** [comment]
Looks good otherwise and thanks for cleaning up all-packages.nix

**Comment by leenaars** [comment]
Thanks for the style suggestions, I removed the empty lines in default.nix and the period in the description.

**Comment by pSub** [comment]
Please squash the commit into two commits (one for the game and the other for the refactoring) and name then according to https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md. Otherwise its ready for merge.

**Comment by pSub** [comment]
Ah and please resolve the conflicts.

**Comment by leenaars** [comment]
Created new pull atomic request.

**Pull request closed without merging**